### ISSUE TITLE
Database Exception - Numeric value out of range

### ISSUE THREAD

**Issue by jalendport** [spicywebau/craft-neo#194]
#### Description
I have a Neo block I'm trying to delete. I've verified it has no child blocks, is a child of no other blocks, and has no fields. Basically, it's a pretty clean block. It used to be my `row` block (basically the only top-level block) and all my other blocks used to be child blocks. I've since made my other blocks top-level and basically stripped this `row` block clean. I'm 99% sure none of my entries are still 
"actively" using this block. However, when I go to delete it, I'm receiving the following error:

```
Database Exception – yii\db\Exception
SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '`mysite`.`structureelements`.`level` - 2'
The SQL being executed was: UPDATE `structureelements` SET `level`=`level`-2, `dateUpdated`='2019-03-13 14:07:33' WHERE ((`lft` >= 8) AND (`rgt` <= 9)) AND (`root`=3846)
Error Info: Array
(
    [0] => 22003
    [1] => 1690
    [2] => BIGINT UNSIGNED value is out of range in '`mysite`.`structureelements`.`level` - 2'
)
↵
Caused by: PDOException
SQLSTATE[22003]: Numeric value out of range: 1690 BIGINT UNSIGNED value is out of range in '`mysite`.`structureelements`.`level` - 2'
in /Users/jalendport/Projects/mysite/dev/vendor/yiisoft/yii2/db/Command.php at line 1290
```

#### Steps to reproduce
Not sure if this reproducible or not 😬 It could be I'm just experiencing an issue that's very unique to my situation. More than happy to try any debugging steps or whatever may be helpful to you all in trying to figure this out.

#### Other information
- Neo version: 2.2.5
- Craft version: Craft Solo 3.1.17.2
- Is the affected Craft installation multi-site: No

**Comment by ttempleton** [comment]
Thanks for reporting.  This is similar to issue #135, but occurring in a different situation.  I've applied the same fix for that (deleting blocks in reverse to avoid issues with invalid block levels) when deleting a block type and its associated blocks, so that should now be fixed in Neo 2.2.6.

**Comment by ttempleton** [comment]
No worries @jalendport -- though this was similar to #135, it differed in how the error was triggered from a Neo user's perspective, so I'd consider it a different issue (or a reminder that I missed a spot, I suppose 😅)
